client/spa/js/instructor/instructor.controller.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Backbone = require('../vendor/index').Backbone;
+var $ = require('../vendor/index').$;
+var Model = require('./instructor.model');
+var View = require('./instructor.view');
+
+module.exports = Backbone.Controller.extend({
+  routes: {
+    'instructor/:id': 'showInstructor'
```

Comment:
The convention for the route should be instructors/:id. Use the plural to same consistent with the REST api.